ArduCopter/mode_stabilize.cpp

```diff
@@ -55,7 +55,15 @@ void ModeStabilize::run()
     }
 
     // call attitude controller
-    attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(target_roll, target_pitch, target_yaw_rate);
+    if (auto_yaw.mode() != AUTO_YAW_FIXED
+        || target_yaw_rate > 200 || target_yaw_rate < -200) { // range of target_yaw_rate is about -10000 to 10000
```

Comment:
Comment and coded limits don't seem to match up?

Reply:
This was really useful when testing, so the RC operator could take over control.  After one CONDITION_YAW command, the RC operator lost control. We'll update the comment.